BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -450,7 +450,9 @@
         <%= name %> <%% groupCheckbox.('<%= id %>',<%= defaultChecked %>) %>
       </h3>
       <h5 style="font-size : 9.5pt; font-weight: normal; padding-top: 2px;" class="grp-desc">
-        <%= groupDescription %>
+        <% groupDescription.split('\n') do |paragraph|%>
```

Comment:
That code change does not split lines in the rendered page, there are no new `<br>` introduced